src/common/utils.test.js

```diff
@@ -115,4 +124,37 @@ describe('keepKeys', () => {
       expect(actionType.RESET).toBe('HOUSE_CATS__START_THE_RACE__RESET');
     });
   });
+
+  describe('getDateWithDaysOffset', () => {
```

Comment:
I think what makes your `getDateWithDaysOffset` interesting and non-trivial is that it wraps to the previous month/year correctly, which isn't obvious from reading its definition. Consider adding a test case (or just changing your existing one) to demonstrate that `getTodayWithDaysOffset` works on, say, January 1st.

Reply:
That's a good point. I'll add those those tests. Thanks.